### PR TITLE
AWS: simple stockout checker utility

### PR DESCRIPTION
**Pull request by justinsb**
Tries to launch a c4.large in each region, and detects the
InsufficientInstanceCapacity error.

Related to #8170

### CODE REVIEW

File: experiment/aws-stockout/main.go [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+
+var imageName = flag.String("image-name", "amzn-ami-hvm-2017.09.1.20180115-x86_64-gp2", "name of AMI to launch")
+var imageOwner = flag.String("image-owner", "137112412989", "owner of AMI to launch")
+
+var instanceType = flag.String("instance-type", "c4.large", "instance type to check")
+
+var tagKey = "k8s.io/test-infra/check-stockout"
+
+func main() {
+	flag.Set("alsologtostderr", "true")
```
Comment (BenTheElder) [inline experiment/aws-stockout/main.go]: ewww!

this is in experiment so not blocking, but reasons to not use glog. test-infra generally uses logrus instead :-)

**Comment by BenTheElder** [comment]
needs gofmt, otherwise seems mostly fine.

**Comment by justinsb** [comment]
Updated gofmt - whoops!

I'll try logrus for the next one :-)

**Comment by BenTheElder** [comment]
> I'll try logrus for the next one :-)

:+1: you should, it's pretty nice.

**Pull request merged**